opendistro-elasticsearch-jdbc.release-notes

```diff
@@ -1,4 +1,10 @@
-## 2019-10-29, Version 1.3.0 (Current)
+## 2019-12-11, Version 1.4.0 (Current)
+
+### Changes
+
+* Add documentation for connecting Tableau with Open Distro for Elasticsearch using JDBC Driver
+
```

Comment:
The IP type was supported in JDBC right now, could you check if it was in the between of od1.3 and od1.4?

Reply:
The issue on GitHub was opened on 12/5, I'm pretty sure that is between 1.3 and 1.4